app/controllers/status_controller.rb

```diff
@@ -0,0 +1,3 @@
+class StatusController < ::ApplicationController
+  include Insights::API::Common::Status::Api
```

Comment:
@lindgrenj6 it'll raise an exception (it uses PG::Connection)

Reply:
Yep I'm a derp, forgot this app doesn't have a db. followup here: https://github.com/RedHatInsights/topological_inventory-ingress_api/pull/109

we can have the liveness probe time out after 2 seconds so if it takes longer it'll still fail.